mortgage/E2E.ipynb

```diff
@@ -4,7 +4,41 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Imports and Helper Functions"
+    "# Mortgage Workflow\n",
+    "\n",
+    "## The Dataset\n",
+    "The dataset used with this workflow is derived from [Fannie Mae’s Single-Family Loan Performance Data](http://www.fanniemae.com/portal/funding-the-market/data/loan-performance-data.html) with all rights reserved by Fannie Mae. This processed dataset is redistributed with permission and consent from Fannie Mae.\n",
+    "\n",
+    "To acquire this dataset, please visit [RAPIDS Datasets Homepage](https://rapidsai.github.io/datasets/)\n",
```

Comment:
This should use the new link https://rapidsai.github.io/demos/datasets/mortgage-data

Reply:
Updated in the notebook and the readme. Can you confirm this is sufficient?